src/components/Layout/Layout.test.js

```diff
@@ -1,21 +1,46 @@
 import React from "react";
-import { shallow } from "enzyme";
-
+import { BrowserRouter as Router } from "react-router-dom";
+import { mount } from "enzyme";
+import configureStore from "redux-mock-store";
+import { Provider } from "react-redux";
+import dataDump from "testing/complete-redux-store-dump";
 import Layout from "./Layout";
 
+const mockStore = configureStore([]);
 describe("Layout", () => {
   it("renders without crashing and matches snapshot", () => {
-    const wrapper = shallow(<Layout />);
-    expect(wrapper).toMatchSnapshot();
+    const store = mockStore(dataDump);
+    const wrapper = mount(
+      <Provider store={store}>
+        <Router>
```

Comment:
Should we always be using `MemoryRouter` in tests to avoid any potential issues with routes changing? 🤷‍♂

Reply:
Haven't seen any issues so far - if so, we can make the change then.